Library/Homebrew/livecheck/strategy.rb

```diff
@@ -58,6 +58,9 @@ module Strategy
       # `curl` arguments used in `Strategy#page_content` method.
       PAGE_CONTENT_CURL_ARGS = ([
         "--compressed",
+        # Allow the return of an error code on exit, so that we can retry
+        # with a fake UA string (e.g. for Cloudflare-protected sites)
```

Comment:
```suggestion
        # Return an error when the HTTP response code is 400 or greater but
        # continue to return body content
```

I plan to eventually remove the logic to automatically retry the request with a different user agent in the future (in favor of explicitly specifying a different user agent in a `livecheck` block). This won't happen until after I've created a PR for my work to allow setting the user agent in a `livecheck` block (the work has been done for a while, I just need to clean it up and ship it).

In the interim time, there's something to be said for avoiding the reference to the retry logic in this comment, so I don't have to remember to rewrite it then. The suggestion above borrows some of the language from the [`curl` manpage](https://curl.se/docs/manpage.html#-f).

Reply:
Okay, cool! I also like the idea of using similar phrasing to `curl(1)`. I’ve made the changes